config/initializers/sso_cookies.rb

```diff
@@ -1,5 +1,7 @@
 # https://github.com/rails/rails/blob/master/actionpack/lib/action_dispatch/middleware/cookies.rb
 
+# Note: when we upgrade to Rails 5, everyone will be logged out of SSO.
```

Comment:
If you are explicitly setting the cipher and you are ok with continuing to be on CBC forever, they won't be logged out.